content/docs/pipelines/deployment-alternatives.md

```diff
@@ -0,0 +1,140 @@
++++
+title = "Pipelines Deployment Alternatives"
+description = "Alternative approaches to deploy lightweight Kubeflow Pipelines to a cluster"
+weight = 15
++++
+
+As an alternative to [deploying Kubeflow](/docs/started/getting-started/#installing-kubeflow) as a
```

Comment:
Is this GCP only? The instructions below say to use the inverse proxy and I believe that is GCP only?

Reply:
Thanks for the question. I don't know enough context about this.

@IronPan Can you answer after coming back to work?

Reply:
@IronPan reminder, do you know the answer?

Reply:
yes inverse proxy is gcp only.

Reply:
@IronPan I see, shall we rename this document to deployment alternative for GCP? Will it be possible to be extended to work in other cloud platforms?

Reply:
discussed, I will rephrase it to target GCP only